examples/simple/src/comments/PostQuickCreateSaveButton.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
```

Comment:
why do we need a custom save button?

Reply:
Because of [this](https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/button/SaveButton.js#L123)

And [this](https://github.com/marmelab/react-admin/blob/master/packages/ra-core/src/reducer/admin/saving.js#L12)

In a nutshell, the save button display a `CircularProgress` if it reads `saving` from the reducer which is only set when our CRUD actions are dispatched. 

In our case, we are using the `dataProvider` directly so that we can get its result. Hence, we don't dispatch those CRUD actions and cannot set the `saving` prop on the button

Reply:
Can't we make the save button reusable by getting the `saving` state not in `<SaveButton>` but in `<SimpleForm>`?

Reply:
Good idea :+1: